### PR TITLE
Added More To The Rough Story

### PR DESCRIPTION
**Pull request by ArvickC**
Added More to The "roughstory.md" file:

Fixed Indenting
Removed ":"
Capitalized Fix
More Story

### CODE REVIEW

File: RoughStory.md [hunk 1]
```diff
@@ -5,8 +5,23 @@
...
-	*Voice Line*: Hmm, I need wood to repair the roof *Player Takes Controll*
+	*Player Voice Line* Hmm, I need wood to repair the roof *Player Takes Controll*
 	
-	*player gets wood (amount undecided)*  *Voice Line*: If I craft ladders I could get to the top of the roof!
+	*player gets wood (amount undecided)*  *Player Voice Line* If I craft ladders I could get to the top of the roof!
 	
-	*Player gets axe "Dull Axe" Iron, Low Dur., Low Damge* *Voice Line* Sweeeeet
+	*Player gets axe "Dull Axe" Iron, Low Dur., Low Damge* *Player Voice Line* Sweeeeet
+	
+	You then have the freedom to walk around, but the main goal is to find the path. Along the path you find a person giving you the "Strange Book". He says, "In this book you will discover all the skills crafting recipes and wonders of the world!". You are hesitent to take the book... but you do and move on with your day. Once you get to the village, you seem to feel that everyone is trying to walk away from you.
```
Comment (WinstonLuo07) [inline RoughStory.md:14]: personally, I think the other way to get the book is better. Finding a house leads to some questions; who's house is this, why is there a book here. While the person just seems kind of bland.

**Review by WinstonLuo07 — CHANGES_REQUESTED** [review]
Please change the person in the story

File: RoughStory.md (hunk 1 above)
Comment (ArvickC) [reply RoughStory.md:14]: Hmm, Good point! After this meeting ill do that change!

Comment (ArvickC) [reply RoughStory.md]: Added Update

**Pull request merged**